docs/source/getstarted.rst

```diff
@@ -31,8 +31,9 @@ Starting Up the SDK
 2. Open the Cozmo app on the phone. Make sure Cozmo is on and connected to the app via WiFi.
 3. Tap on the gear icon at the top right corner to open the Settings menu.
 4. Swipe left to show the Cozmo SDK option and tap the Enable SDK button.
-5. On the computer, open a Terminal (macOS/Linux) / Command Prompt (Windows) window. Type ``cd cozmo_sdk_examples`` where *cozmo_sdk_examples* is the name of the directory you extracted the SDK examples into and press Enter. The SDK example scripts can be downloaded from the :doc:`Downloads page </downloads>`.
-6. Now you can run any of the example programs.
+5. Make sure the SDK examples are downloaded from the :doc:`Downloads page </downloads>`.
+6. On the computer, open Terminal (macOS/Linux) or Command Prompt (Windows) and navigate to the folder containing that program. For example, if the cozmo_sdk_examples folder is saved to your desktop, type ``cd Desktop/cozmo_sdk_examples``
```

Comment:
If we are having the cd statement, we should have a period after the ``cd Desktop/cozmo_sdk_examples``. We should also have a period at the end of the sentence "For Windows, replace forward slash...." However, as Mark mentioned earlier, avoiding cd'ing into multiple folders can prevent us having to discuss this. For sure some Windows users are going to run the cd command listed here and get an error.